vendor/smart-parks/opencollar.yaml

```diff
@@ -0,0 +1,24 @@
+name: OpenCollar
+description: First generation of OpenCollar devices. A collection of advanced devices designed for wildlife protection.
+
+# Hardware versions (optional, use when you have revisions)
+hardwareVersions:
+  - version: '2.3'
+    numeric: 1
+  - # Firmware version 2.6
```

Comment:
please follow the order in this line, you delete `firmwareVersion:` line check example file: